temperature/crud.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import aiohttp
+from dotenv import load_dotenv
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import Session
+
+from city.models import City
+from temperature import schemas, models
+
+load_dotenv()
+
+API_KEY = os.getenv("API_KEY")
+URL = os.getenv("URL")
+
+
+async def fetch_temperature(city_name: str) -> float:
+    async with aiohttp.ClientSession() as session:
+        async with session.get(f"{URL}key={API_KEY}&q={city_name}") as response:
+            data = await response.json()
+            return data["current"]["temp_c"]
```

Comment:
Consider adding error handling for the API request. If the request fails or the response does not contain the expected data, it could lead to runtime errors. Implementing try-except blocks or checking the response status code would be beneficial.